assignments/callbacks.js

```diff
@@ -19,30 +19,85 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
     console.log(first)
   });
 
-*/
+
 
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+
+return cb(arr.length);
+
 }
+getLength(items, function(length) {
+console.log(length)
+});
+
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
-}
 
+  return cb(items.length - 1);
+
+}
+last(items, function(con) {
+  console.log(con)
+});
+// sumNums adds two numbers (x, y) and passes the result to the callback.
 function sumNums(x, y, cb) {
-  // sumNums adds two numbers (x, y) and passes the result to the callback.
+  cb(x,y);
```

Comment:
you need a return statement here, and remember you want to pass into the callback function a single value that will be x plus y. this would look more like `return cb(x+y)` since that will be adding the two values together before the callback function actually runs